test/distributed/index_map_kernels.cpp

```diff
@@ -388,3 +406,256 @@ TEST_F(IndexMap, GetLocalWithCombinedIndexSpaceWithInvalidIndexSameAsRef)
 
     GKO_ASSERT_ARRAY_EQ(result, dresult);
 }
+
+
+TEST_F(IndexMap, GetGlobalWithLocalIndexSpaceSameAsRef)
```

Comment:
Same here. I think just the test with both invalid and valid indices should be enough because the kernel you are calling is the same, and the code path in both cases is the same except for the bounds check.

Reply:
I think this is not resolved yet ?